build.gradle

```diff
@@ -509,6 +513,37 @@ task runProductionAutoTestServer(type: JavaExec, dependsOn: [remapJar, remapTest
 	}
 }
 
+def addPomMetadataInformation(Project project, MavenPom pom) {
+	def modJsonFile = project.file("src/main/resources/fabric.mod.json")
+	if (!modJsonFile.exists())
+		modJsonFile = project.file("src/client/resources/fabric.mod.json")
+	def modJson = new JsonSlurper().parse(modJsonFile)
+	pom.name = modJson.name
+	pom.url = "https://github.com/FabricMC/fabric/tree/${grgit.head().abbreviatedId}/${project.rootDir.relativePath(project.projectDir)}"
```

Comment:
I would just hard code the URL to `https://github.com/FabricMC/fabric`, as this wont work if someone tries to build it outside of a github repo. (Such as when you just download the zip from github)